class/gitlab-runner.yml

```diff
@@ -16,6 +16,7 @@ parameters:
         helm_params:
           name: gitlab-runner
           namespace: ${gitlab_runner:namespace}
+          api_versions: monitoring.coreos.com/v1
```

Comment:
Personal preference: I'd probably go with enabling monitoring by default, since there should always be a prometheus-operator available on properly configured Project Syn clusters.